doc/man/scons.xml

```diff
@@ -3329,7 +3328,7 @@ with this configuration context.
 However, you can create a new
 Configure
 context to perform additional checks.
-Only one context should be active at a time.</para>
+Only one context may be active at a time.</para>
```

Comment:
Did we add logic to explicitly disallow more than one? If so a note that doing so would cause an error would be a useful addition

Reply:
It's caused an error for a while (apparently?). You made a change recently to actually put out an error message that tells you why (03dbdfc919),

Reply:
Can you add something like you'll see this error if you violate this rule?

Reply:
Which reminds me - not part of this PR specifically - manpage shows none of the errors or warnings scons could generate.